Project21/Project21/Assignment 3.c

```diff
@@ -0,0 +1,46 @@
+////// ------------------------------------------------------------------------------------------------
+//// Assignment 3 ~ A training program for new joiners at Metamation, Batch - July 2024.
+//// Copyright (c) Metamation India.
+//// ------------------------------------------------------------------
+//// Assignment 3
+//// Decimal to Binary conversion and Decimal to HexaDecimal conversion
+//// ------------------------------------------------------------------------------------------------
+#include<stdio.h>
+#define INT_SIZE sizeof(int) * 8
+#define HEX_SIZE 8
+

```

Comment:
Give comments above the function between <summary> tag like ///< summary>..comments..///< /summary>. Also apply this wherever necessary.